collect_app/src/main/java/org/odk/collect/android/utilities/MediaUtils.java

```diff
@@ -613,6 +613,16 @@ public static boolean isGooglePhotosUri(Uri uri) {
                 .getAuthority());
     }
 
+    /**
+     * @param uri The Uri to check
+     * @return Whether the Uri authority is Google Drive.
+     */
+    private static boolean isGoogleDriveDocument(Uri uri) {
```

Comment:
This is the commit that breaks the build. You can `git checkout 4167db49daa9cb24f197beb1d968bb9be50d6501` to verify this. In general, make sure you compile before each commit -- each commit should leave the build green! Are you using an IDE? In Android Studio, I immediately see all the compilation errors in this file flagged.